### PR TITLE
Update linting action

### PR DESCRIPTION
**Pull request by avalor1**
Old action got replaced with 'ansible-lint'

### CODE REVIEW

**Review by schurzi — CHANGES_REQUESTED** [review]
Can you please solve the "unrelated" lint errors? I suspect they are new because of the lint update.

**Comment by avalor1** [comment]
Yes one came with the update, the line to long popped up today before the update of me. Hoped to fix them both

**Comment by avalor1** [comment]
The .yamllint file was not used due to the .cfg extension. After fixing this i had to fix some more linting errors. Now all checks are green again

File: meta/runtime.yml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 ---
-requires_ansible: ">=2.9.10"
+requires_ansible: "<=2.17.0"
```
Comment (schurzi) [inline meta/runtime.yml:2]: Version smaler that the current one? are you expecting this to break with a nex Ansible release?

Comment (avalor1) [reply meta/runtime.yml:2]: No but that is the highest stable version which is covered in our tests. Also the 2.9.10 also works but linter says it should be used a supported version in this variable. You mean we should use >=2.15.0?

Comment (schurzi) [reply meta/runtime.yml:2]: I don't link this change. That would imply that we need to update this with every new Ansible version. I would only like to update things here if stuff actually breaks.

Maybe disable this rule like we did in dev-sec: https://github.com/dev-sec/ansible-collection-hardening/blob/master/.config/ansible-lint.yml#L20

Comment (avalor1) [reply meta/runtime.yml]: Yes, i think that would be the best. Thank you, I changed it

**Review by schurzi — APPROVED** [review]

**Pull request merged**